pkg/tanka/inline.go

```diff
@@ -17,6 +18,17 @@ import (
 // Kubernetes resources are expected at the `data` key of this very type
 type InlineLoader struct{}
 
+func (i *InlineLoader) Test(base string) (bool, error) {
+	_, err := os.Stat(filepath.Join(base, spec.Specfile))
```

Comment:
The check for a spec file should go in the StaticLoader, I suggest to use the Peek function to check for Inline here.